src/components/Home/Home.module.css

```diff
@@ -27,7 +26,7 @@
     font-weight: bold
 }
 
-.homePageSecondaryButton{
+/* .homePageSecondaryButton{
```

Comment:
Let's remove this commented css @vishwaikon. Check whether the other CSS classes are also being used if not we can remove the CSS file altogether

Reply:
@anjula-sack, Sure I'll. Thought of keeping a few CSS classes till the remaining parts are completed to prevent conflicts.

Reply:
yeah we can remove the unused css.